ansible/doc/README.testbed.Setup.md

```diff
@@ -96,15 +162,23 @@ Once you are in the docker container, you need to modify the testbed configurati
         ```
 
 - VMs
-    - Download [vEOS image from Arista](https://www.arista.com/en/support/software-download).
+    - Download [vEOS-lab image from Arista](https://www.arista.com/en/support/software-download).
     
     - Copy these image files to `~/veos-vm/images` on your testbed server:
         - `Aboot-veos-serial-8.0.0.iso`
-        - `vEOS-lab-4.20.15M.vmdk`
+        - `vEOS-lab-4.25.0F.vmdk`
```

Comment:
my recommendation is not change this until you have validated all the tests with this vEOS image?

Reply:
i need to revert this, tested and did not work.